### PR TITLE
Compute hmac

### LINKED ISSUE #46: Exercise 21
Compute a Hashed Message Authentication Code (HMAC)

### PR DESCRIPTION
**Pull request by lrsmit**
Resolves #46

### CODE REVIEW

File: set4/ex21.tex [hunk 1]
```diff
@@ -0,0 +1,30 @@
+\section*{Exercise 21}
+\subsection*{Computing a Hashed Message Authentication Code (HMAC)}
+The chosen HMAC key is "ruben-smit". With the $security.png$ file the calculated HMAC will be \\
```
Comment (mr-frying-pan) [inline set4/ex21.tex]: 1. Quotes written like this look ugly. This is better:
```suggestion
The chosen HMAC key is ``ruben-smit''. With the $security.png$ file the calculated HMAC will be \\ 
```
2. $ signs open and close math environment. I am sure that's not what you want. Alternatives are `\texttt{...}` for monospace font and `\textit{...}` for italics

File: set4/hmac/ex21.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+import sys
+import argparse
+import hmac
```
Comment (mr-frying-pan) [inline set4/hmac/ex21.py]: Are you sure you can use these?

Comment (lrsmit) [reply set4/hmac/ex21.py]: I'm sure about argparse. I'm not sure about hmac. They're both standard libraries though, they come with python.

Comment (mr-frying-pan) [reply set4/hmac/ex21.py]: Well, then let's hope you can :D

**Pull request merged**